src/main/java/com/bina/varsim/fastqLiftover/DWGSIMFastqReader.java

```diff
@@ -44,7 +44,7 @@ public SimulatedRead getNextRead() throws IOException {
         final int start2 = Integer.parseInt(nameFields[nameFields.length - 13]);
         final int start1 = Integer.parseInt(nameFields[nameFields.length - 14]);
         final String chromosome1 = joiner.join(Arrays.copyOfRange(nameFields, 0, nameFields.length - 14));
-        final String chromosome2 = chromosome1;
+        final String chromosome2 = chromosome1; // Marghoob was there a reason for this?
```

Comment:
No specific reason. Might be that someday we will support translocations in which case the two fragments can possibly belong to different contigs.